operators/src/source/ogr_source.rs

```diff
@@ -765,9 +757,9 @@ where
                         Ok(Some(FieldValue::IntegerValue(v))) => Some(f64::from(v)),
                         Ok(Some(FieldValue::StringValue(s))) => f64::from_str(&s).ok(),
                         Ok(Some(FieldValue::RealValue(v))) => Some(v),
-                        Ok(Some(_)) => return Err(Error::OgrColumnFieldTypeMismatch), // TODO: handle other types
                         Ok(None) => None,
-                        Err(_) => None, // TODO: log error
+                        Ok(Some(_)) => error_spec.on_error(Error::OgrColumnFieldTypeMismatch)?,
+                        Err(e) => error_spec.on_error(Error::Gdal { source: e })?, // TODO: handle other types
```

Comment:
move todo back to `Ok(Some(_)) => …`

Reply:
8f95de873564abf89e0b7752302b1ab30ebb97fa